### ISSUE TITLE
jquery-ui-multiselect-widget keycode issue

### ISSUE THREAD

**Issue by devtanu** [ehynds/jquery-ui-multiselect-widget#630]
I'm using similar feature as showed in the link below :
http://www.erichynds.com/jquery/jquery-ui-multiselect-widget/
But if seems jquery-ui-multiselect-widget is not completely keyboard friendly. It's failing some of the accessibility tests. 
I've multiple dropdown fields having multi-slect checkbox options. By pressing Tab, I can move one dropdown to other, by pressing space-bar, I can see the options within the drop-down but there is no keycode/charCode to fire the event for "Check All/Uncheck All" links.
Can somebody here, please share their inputs/ suggestions on this.
TIA

Regards,
Tanmay

**Comment by mlh758** [comment]
I've added shortcuts for those commands: https://github.com/ehynds/jquery-ui-multiselect-widget/wiki/Keyboard-Navigation
